src/libs/DebugUtils.ts

```diff
@@ -612,8 +611,12 @@ function getReasonForShowingRowInLHN(report: OnyxEntry<Report>): TranslationPath
         includeSelfDM: true,
     });
 
-    // When there's no specific reason, we default to isFocused since the report is only showing because we're viewing it
+    // When there's no specific reason, we default to isFocused if the report is only showing because we're viewing it
+    // Otherwise we return hasRBR if the report has errors other that failed receipt
     if (reason === null || reason === CONST.REPORT_IN_LHN_REASONS.DEFAULT) {
+        if (ReportUtils.hasReportErrorsOtherThanFailedReceipt(report, doesReportHaveViolations, transactionViolations)) {
```

Comment:
Let's move this to the outside of reason check and assign reason = hasRBR if ReportUtils.hasReportErrorsOtherThanFailedReceipt is true. I think this way will be clearer

Reply:
In case the reason is HAS_GBR and we have an error, we will display
- GBR : false
- RBR: true
Because of this condition:

https://github.com/Expensify/App/blob/7947b8e25c9c4799f8bec6e3986d129db06a2939/src/pages/Debug/Report/DebugReportPage.tsx#L68

So that, we also need to display a suitable reason to explain why this report appears on LHN

Reply:
Nice catch @DylanDylann ! 😄